tests/unit/test_blob.py

```diff
@@ -2875,9 +2873,9 @@ def _do_resumable_upload_call1(
     ):
         # Second mock transport.request() does sends first chunk.
         if size is None:
-            content_range = "bytes 0-{:d}/*".format(blob.chunk_size - 1)
+            content_range = f"bytes 0-{blob.chunk_size - 1:d}/*"
```

Comment:
I think the `:d` format specifier is not needed anymore

```suggestion
            content_range = f"bytes 0-{blob.chunk_size - 1:}/*"
```